nix/overrides/bazel/nix-hacks.patch

```diff
@@ -1,33 +1,35 @@
 diff -Naur a/src/main/java/com/google/devtools/build/lib/rules/repository/RepositoryDelegatorFunction.java b/src/main/java/com/google/devtools/build/lib/rules/repository/RepositoryDelegatorFunction.java
---- a/src/main/java/com/google/devtools/build/lib/rules/repository/RepositoryDelegatorFunction.java	1980-01-01 00:00:00.000000000 -0500
-+++ b/src/main/java/com/google/devtools/build/lib/rules/repository/RepositoryDelegatorFunction.java	2018-01-18 08:17:22.420459162 -0500
-@@ -287,21 +287,8 @@
-           markerData.put(key, value);
-         }
-       }
--      boolean result = false;
--      if (markerRuleKey.equals(ruleKey)) {
--        result = handler.verifyMarkerData(rule, markerData, env);
--        if (env.valuesMissing()) {
+--- a/src/main/java/com/google/devtools/build/lib/rules/repository/RepositoryDelegatorFunction.java	2019-06-12 20:39:37.420705161 -0700
```

Comment:
Now that we are on `nixpkgs-unstable` we might be able to just grab the file from there.

Reply:
You mean by something like `"${pkgs.path}/path/to/patch"`? Or you mean the whole Bazel build? We still apply a few patches compared to upstream nixpkgs.